src/test/java/com/mael/ttt/BoardTest.java

```diff
@@ -39,7 +47,7 @@ public void resetsBoard() {
     @Test
     public void getsTheEmptyCellIndexes() {
         board.setCell(1, PLAYER);
-        List<Integer> expected = asList(2, 3, 4, 5, 6, 7, 8, 9);
+        List<Integer> expected = getCellIndexesStartingOn(2);
```

Comment:
In the BoardTest class, that was the only method that needed to change if the size of the board changed. I could add the same test for the 4x4 board and just create the board inside the test:

```java
@Test
public void getsTheEmptyCellIndexes() {
    board = new Board(4);
    board.setCell(1, PLAYER);
    List<Integer> expected = asList(2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16);
    assertEquals(expected, board.getEmptyCellIndexes());
}
```

And I wouldn't need to parameterize the test class. And the board checker test class works without even changing a comma for any board size. 

HOWEVER.

All the test methods should pass no matter the board size. So, if I wanted to check that the tests pass for `size = 4`, I would have to go to the board test class, change the size, run the class. Then do the same for the checker. Then I introduce some feature, and I want to run the test class for `size = 3` again. I have to go to the board test class, change the size, run the test, then go to the board checker test class, change the size to 3, run the test again, then go back to the board test class, change the size to 4, run the test, then go to the checker, change the size to 4... 

:boom: BRAIN EXPLODES :boom: 

So, I added this parameterization just automates the process.